src/main/java/org/apache/maven/shared/verifier/Verifier.java

```diff
@@ -1599,11 +1576,28 @@ public void setCliOptions( List<String> cliOptions )
         this.cliOptions = cliOptions;
     }
 
+    /**
+     * Add a command line argument, each argument must be set separately one by one.
+     * <p>
+     * <code>${basedir}</code> in argument will be replaced by value of {@link #getBasedir()}
```

Comment:
Note that this comment is actually wrong because the value isn't replaced here, but only at execute.